tests/components/mqtt/test_util.py

```diff
@@ -50,9 +53,9 @@ async def test_async_create_certificate_temp_files(
     # Make sure certificate temp files are recovered
     if file_path:
         # Overwrite content of file (except for auto option)
-        file = open(file_path, "wb")
-        file.write(b"invalid")
-        file.close()
+        with open(file_path, "wb") as file:
+            file.write(b"invalid")
+            file.close()
```

Comment:
This does blocking I/O

Reply:
Should I rewrite this code part?

Reply:
I'd delegate the I/O via `await hass.async_add_executor_job`

Reply:
Right